cypress/integration/e2e/management-tab/edit-worker.spec.ts

```diff
@@ -51,4 +52,25 @@ describe("Tab management", () => {
       cy.get('[data-cy="input-civil-time"] input').clear({ force: true }).type("123");
     });
   });
+
+  const testWorker = "Pielęgniarka 2";
```

Comment:
Czy to nie jest pozostałość z [TASK-153](https://github.com/Project-Summer-AI-Lab-Glider/nurse-scheduling-problem-frontend/pull/368/commits/05b276ab040d49be4942b62a6bdb2d0bbc3fef5e)?

Reply:
Tak, to relikt z  TASK-153, usunę konflikty przy merdżu.